src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/MessageStrings.resx

```diff
@@ -135,6 +135,9 @@
   <data name="FailedToEditStartup" xml:space="preserve">
     <value>Scaffolding failed to edit Startup class to register the new Context using Dependency Injection.</value>
   </data>
+  <data name="ModelTypeCouldNotBeAdded" xml:space="preserve">
+    <value>Could not add Model type '{0}' to DbContext '{1}'. If '{1}' is in a dependency, make sure it has a DbSet property for '{0}'.</value>
```

Comment:
I agree. Would it make sense to just say:
`Please make sure that '{1}' has a DbSet property for '{0}'.` ?